index.js

```diff
@@ -1,13 +1,36 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header") 
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
+header.style.color = 'red'
 
 /***** Deliverable 3 *****/
-// console.log("PLAYERS array looks like this:", PLAYERS)
+console.log("PLAYERS array looks like this:", PLAYERS)
 
 
-/***** Deliverable 4 *****/
\ No newline at end of file
+
+var parentElm = document.querySelector(".player-container")
+
+PLAYERS.forEach((player) => {
+    let divPlayer = document.createElement('div')
+    divPlayer.innerHTML = `
+                    <div class='player' data-number="${player.number}"> 
+                    <h3>
+                        ${player.name} (<em>${player.nickname}</em>)
+                    </h3>
+                    <img width = 300px src=${player.photo} alt="${player.name}"/>
+                    </div>
+                    `
+    parentElm.appendChild(divPlayer)
+
+    
+})
+
+
+/***** Deliverable 4 *****/
+
+// [attr=value]
+document.querySelector(".player-container [data-number='7']").remove()
```

Comment:
Nice! This will work. If you want to shorten up the CSS selector a bit, you can also just do:

```js
document.querySelector("[data-number='7']").remove()
```